pvw/server/constants.py

```diff
@@ -0,0 +1,327 @@
+### From models.py
```

Comment:
This won't make much sense in the future and we don't really care where it came from.
```suggestion
```